annotated-projects/COMBINEDPX0000001/COMBINEDPX0000001.idf.tsv

```diff
@@ -0,0 +1,34 @@
+MAGE-TAB Version	1.1
+Investigation Title	Reanalysis of multiple ProteomeXchange submissions cell line A549, H1975, H446, H69
+Experiment Description	Reanalysis of multiple ProteomeXchange submissions cell line A549, H1975, H446, H69
+
+Date of Experiment	2021-02-10
+Public Release Date	2021-02-10
+
+Protocol Name	P-MTAB-Sample-PXD000312	P-MTAB-Data-PXD000312
```

Comment:
Is there a problem with the PXID in the `Protocol Name?` There is no the project ID in the `Comment[ProteomeXchange accession number]`

Reply:
You are rigth.